indra/newview/llviewerobject.h

```diff
@@ -28,6 +28,7 @@
 #define LL_LLVIEWEROBJECT_H
 
 #include <map>
+#include <unordered_map>
```

Comment:
this intentional?

Reply:
The Mac build failed without it, claiming there's no such animal as `std::unordered_map`.

Reply:
yeah understandable.  I'm curious how this ever worked without it

Reply:
I won't even deplore our habits as coders, because the C++ language doesn't incentivize checking for the right `#include` every time we reference a new function or class. If our new code compiles, that's good enough...

So (a) many of our .cpp files rely on indirect header-file inclusion, subject to unexpected breakage if the set of `#includes` changes either in the .cpp file or in one of the headers it drags in; and (b) we doubtless have many `#includes` that are no longer needed.

Presumably some IDEs can tag every symbol with the header file declaring it, recognizing whether the current .cpp file `#include`s that same header (possibly redundantly), and can identify directly-included headers none of whose declarations are referenced ... but until such tooling becomes universal, I don't expect us all to suddenly become more meticulous about header usage.